infrastructure/staging/dashboard.tf

```diff
@@ -0,0 +1,85 @@
+# dds-dashboard application
+module "dashboard" {
+  source = "../modules/ecs"
+
+  name        = "iiif-builder-dashboard"
+  environment = local.environment
+  vpc_id      = local.vpc_id
+
+  docker_image   = "${data.terraform_remote_state.common.outputs.dashboard_url}:staging"
+  container_port = 80
+
+  cpu    = 256
+  memory = 512
+
+  ecs_cluster_arn                = aws_ecs_cluster.iiif_builder.arn
+  service_discovery_namespace_id = data.terraform_remote_state.common.outputs.service_discovery_namespace_id
+  service_subnets                = local.vpc_private_subnets
+  service_security_group_ids     = [data.terraform_remote_state.common.outputs.staging_security_group_id, ]
+
+  healthcheck_path = "/management/healthcheck"
+
+  lb_listener_arn = module.load_balancer.https_listener_arn
+  lb_zone_id      = module.load_balancer.lb_zone_id
+  lb_fqdn         = module.load_balancer.lb_dns_name
+
+  listener_priority = 20
+  hostname          = "dds-stage"
+  domain            = local.domain
+  zone_id           = data.aws_route53_zone.external.id
+
+  port_mappings = [{
+    containerPort = 80
+    hostPort      = 80
+    protocol      = "tcp"
+  }]
+
+  secret_env_vars = {
+    ConnectionStrings__Dds                = "iiif-builder/staging/ddsinstrumentation-connstr"
+    ConnectionStrings__DdsInstrumentation = "iiif-builder/staging/dds-connstr"
+    AzureAd__TenantId                     = "iiif-builder/staging/azuread-tenantid"
+    AzureAd__ClientId                     = "iiif-builder/staging/azuread-clientid"
+    Storage-Production__ClientId          = "iiif-builder/common/storage/clientid"
+    Storage-Production__ClientSecret      = "iiif-builder/common/storage/clientsecret"
+    Dlcs__ApiKey                          = "iiif-builder/staging/dlcs-apikey"
```

Comment:
Can these be in /common/ too? They are the same for both envs, like the storage creds

Reply:
(DLCS creds)

Reply:
I went with the DLCS creds being in `/staging/` to allow us to use a different customer for staging in the future, should the need arise.

Reply:
OK, fair enough